lib/1.4/dml-builtins.dml

```diff
@@ -714,7 +714,13 @@ template attribute is _conf_attribute {
 Four templates are used to create a simple checkpointable attribute with
 standard types, storing the attribute value in a member `val`, and
 providing default implementations of methods `get` and `set`
-accordingly:
+accordingly. These default implementations make use of corresponding
+methods <tt>get_val() -> (<em>type</em>)</tt> and
+<tt>set_val(<em>type</em> val) throws</tt>, which can be overridden in order to
+provide custom get/set logic in a more ergonomic way compared to overriding
+`get`/`set` directly.
```

Comment:
Wondering if `get_val()` and `set_val()` should also be mentioned in the language section of the DML 1.4 reference manual. It currently states:
```
Four standard templates are provided for attributes: `bool_attr`, `int64_attr`,
`uint64_attr` and `double_attr`. They provide overridable `get` and `set`
methods, and store the attribute's value in a session variable named `val`,
using the corresponding type.
```